scripts/emulations/README.md

```diff
@@ -0,0 +1,23 @@
+# Add Validator
+
+This script will help us in emulating various feature in blockchain programmatically. 
+
+<br />
+
+## Requirements
+[NodeJs (v14.17.0 lts)](https://nodejs.org/en/download/ "NodeJs (v14.17.0 lts)")
+
+<br />
+
+## Configure app
+Open `config.json` and update the with appropriate values to emulate any emulations.
```

Comment:
`and update it with the appropriate values to emulate`

Reply:
done.